pkg/secrets/config/config.go

```diff
@@ -138,51 +165,53 @@ func ValidateSecretsProviderSettings(envAndAnnots map[string]string) ([]error, [
 	}
 
 	envStoreType := envAndAnnots["SECRETS_DESTINATION"]
-	annotStoreType := envAndAnnots["conjur.org/secrets-destination"]
+	annotStoreType := envAndAnnots[SecretsDestinationKey]
 	storeType := ""
+	var err error
 
 	if annotStoreType == "" {
```

Comment:
Your call on this, but since you're rewriting some of this already.... Golang coding style recommends using `switch` block rather than `if`/`else if`/`else` block. So this section would have the form:
```
    switch {
        case annotStoreType == "":
            // Do things
        case validStoreType(annotStoreType):
            // Do other things
        default:
            // Log an error
    }
```

Reply:
done